src/Chapter2.hs

```diff
@@ -500,7 +519,11 @@ True
 >>> isThird42 [42, 42, 0, 42]
 False
 -}
-isThird42 = error "isThird42: Not implemented!"
+
+isThird42 :: [Int] -> Bool
+isThird42 (x:(y:42:xs)) = True
+isThird42 (_:xs) = False
```

Comment:
That's the correct implementation! :gem: 

Usually, when variable names don't matter, it's recommended to use underscores `_` instead of them to indicate in the code, that those variables are not used. And, you also can remove a few extra parenthesis. So, something like this should work and look a bit more cleaner :slightly_smiling_face: 

```suggestion
isThird42 (_ : _ : 42 : _) = True
isThird42 (_ : _) = False
```